robottelo/ui/locators/base.py

```diff
@@ -2631,6 +2635,22 @@
         By.XPATH,
         "//input[@value='Check Connection']"
     ),
+    "insights.actions.export_csv": (
+        By.XPATH,
+        "//button[normalize-space(.)='Export CSV']"
+    ),
+    "insights.actions.issues_count": (
+        By.XPATH,
+        "//span[normalize-space(.)='Issues']/../div[@class='num']"
+    ),
+    "insights.help.general_info": (
+        By.XPATH,
+        "//div[@ng-app='RedhatAccessInsights']//h3[1]"
+    ),
+    "insights.help.getting_started": (
+        By.XPATH,
+        "//div[@ng-app='RedhatAccessInsights']//h3[2]"
```

Comment:
explicit index isn't any better than text value in xpath. as fragile as text but also less readable. if there's nothing better to use like class or id, i'd rather prefer text over index. @oshtaier thoughts?

Reply:
Here is the html code of the page.
```html
<div ng-app="RedhatAccessInsights">
    <!----><div ui-view="" class="wrapper in-prod"><div class="section">
    <section>
        <h3>
           General Information
        </h3>
        <p>
          For an overview of the Red Hat Insights service, please refer to the <a href="https://access.redhat.com/insights/info/" target="_blank">general information page</a>.
        </p>
        <h3>
           Getting Started
        </h3>
        <p>
          For help with configuring and enabling the Red Hat Insights service in Satellite 6, please refer to the <a href="https://access.redhat.com/insights/getting-started/satellite/6/" target="_blank">Getting Started Guide</a>.
        </p>
    </section>
</div>
</div>
    <div class="actionbar"><!----></div>
  </div>
```

Reply:
Yeah, that is not best style of locators. Sometimes, that questions the whole idea whether we need to test/address such UI sections at all

Reply:
but anyway 6.3 is not so important at that moment, so let's ignore such facts